<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainProjet</class>
 <widget class="QMainWindow" name="MainProjet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainProjet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QPushButton {
        background-color: #D11A0B;
        color: white;
        border-radius: 20px;
        padding: 0px;
        border: 1px solid #D11A0B;
        font-size: 12px;
    }
    QPushButton:hover {
        background-color: #FFFFFF;
        color: #D11A0B;
    }
    QPushButton:pressed {
        background-color: #FFFFFF;
        color: #D11A0B;
    }
    QTableWidget {
        background-color: #FFFFFF;
        border: 1px solid #dcdcdc;
        gridline-color: #dcdcdc;
    }
    QTableWidget QHeaderView::section {
        background-color: #D11A0B;
        color: #FFFFFF;
        padding: 5px;
        border: 1px solid #D11A0B;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>211</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="ButtonDocuments">
       <property name="text">
        <string>Documents</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonReporting">
       <property name="text">
        <string>Reporting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonParametres">
       <property name="text">
        <string>PARAMETRES</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="ButtonInformation">
       <property name="text">
        <string>Informations base</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonRestriction">
       <property name="text">
        <string>Niveau de restriction</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>411</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image_interface/logo_sodial.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonHygiene">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hygiène</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonQualiter">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Qualité</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image_interface/montagne.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>271</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image_interface/logo_entremont1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>50</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image_interface/logo_euroserum.png</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>ButtonHygiene</zorder>
   <zorder>ButtonQualiter</zorder>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
